includes/definitions/quanta.yaml

```diff
@@ -4,11 +4,14 @@ type: network
 icon: quanta
 over:
     - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+mib_dir:
+    - quanta
 discovery:
     - sysObjectId:
           - .1.3.6.1.4.1.7244
-    -
-      sysObjectId:
```

Comment:
How come this was changed? I know it doesn't look much but moving this does change the way detection is done.